### PR TITLE
Yvonne

### PR DESCRIPTION
**Pull request by YvonneKallberg**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+							Sweden
+						</td>
+					</tr>
+					<tr>
+						<th>Email</th>
+						<td><a href="mailto:datacentre@scilifelab.se">datacentre@scilifelab.se</a>
+						</td>
+					</tr>
+					<tr>
+						<th class="contact-table-th">Bug reporting</th>
```
Comment (talavis) [inline contact.html:39]: Maybe add a comment about issues in the SciLifeLab knowledge models to be reported to our github repository?

Comment (YvonneKallberg) [reply contact.html:39]: So, have info about two issue reports, one general dsw, and one SLL KM?

Comment (talavis) [reply contact.html:39]: Exactly; I don't think they would be happy to get reports about misstakes in our KMs.

Comment (YvonneKallberg) [reply contact.html:39]: Which link do I use to SLL KM?

Comment (talavis) [reply contact.html:39]: https://github.com/ScilifelabDataCentre/dsw-issues

Just like the issues reported directly from inside DSW.

**Pull request merged**